de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoice/model/validator/C_Invoice.java

```diff
@@ -13,11 +13,11 @@
  *
```

Comment:
we are not (yet?) totally consistent on this, but: i suggest as a best practice to have MI-methods in the "home" module of the records they influence. 
Here, it's a C_Invoice change that has an influences on C_Invoice_Candidate, so please move it to a C_Invoice-MI within de.metas.invoicecandiate